master-docker-nonstandard-2/master.cfg

```diff
@@ -923,13 +923,13 @@ c["builders"].append(
 
 c["builders"].append(
     util.BuilderConfig(
-        name="amd64-debian-11-msan",
-        workernames=workers["x64-bbw-docker-msan-clang-debian-11"],
-        tags=["Debian", "quick", "clang-15", "msan"],
+        name="amd64-msan-clang",
```

Comment:
@grooverdan Please check the pre-commit checks and validate-masters tests. Your PR fails those checks. You should check those *before* sending a PR for review.

Reply:
it was disclosed and mentioned as a thing to decide on deployment.